optaplanner-core/src/main/java/org/optaplanner/core/config/domain/ReflectionsWorkaroundClasspathHelper.java

```diff
@@ -38,7 +38,10 @@
 /**
  * Workaround for bug in {@link ClasspathHelper}.
  * This workaround expires once https://github.com/ronmamo/reflections/pull/118 is fixed.
+ * 
+ * @deprecated for removal.
```

Comment:
in favor of Quarkus extension or Spring Boot starter.

Reply:
I actually re-read the comment and this class may no longer be necessary as the PR in question has been merged.